jena-shacl/src/main/java/org/apache/jena/shacl/engine/constraint/QualifiedValueShape.java

```diff
@@ -50,6 +50,22 @@ public QualifiedValueShape(Shape sub, int qMin, int qMax, boolean qDisjoint) {
         this.qDisjoint = qDisjoint;
     }
 
+    public Shape getSub() {
+        return sub;
+    }
+
+    public int getqMin() {
```

Comment:
AutoCase does not work out! And "Q" does work that well either! How about `public int qMin()`?

Reply:
ok (`getQMin()` has its merits, but your call)

Reply:
Ok